timemachine/fe/rest/single_topology.py

```diff
@@ -89,7 +91,7 @@ def rest_region_atom_idxs(self) -> set[int]:
         tends to result in larger REST regions than seem desirable.
         """
 
-        aligned_potentials: list[AlignedPotential] = [
+        aligned_potentials: list[Union[AlignedBond, AlignedAngle, AlignedTorsion]] = [
```

Comment:
nit/later: might be able to clean this up by introducing an abstract `AlignedBondedPotential` that adds `idxs` to the fields

Reply:
A bit torn - I thought about this as well, but was debating between the cost of having yet another class in the inheritance hierarchy.